### PR TITLE
Add tabs component

### PR DESCRIPTION
**Pull request by dine**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -32,9 +32,6 @@ module.exports = {
                 'fade-right-leave-active':
                     'fd-fade-right 0.6s 1 ease-out reverse',
             },
-            borderWidth: {
```
Comment (dine) [inline tailwind.config.js:35]: About this change: https://github.com/funda-frontend/ui/pull/143#discussion_r522785002

File: src/stories/ui-tabs.stories.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    }),
+    template: ` <div>
+                    <ui-tabs>
+                        <template v-for="(tab, i) in tabs">
+                            <ui-tab :selected="selected==tab.value" @click="onClick(tab.value)">{{ tab.label }}</ui-tab>
+                        </template>
+                    </ui-tabs>
+                    <template v-for="(tab, i) in tabs">
+                        <div v-if="selected==tab.value">
+                            {{ tab.value }} content
```
Comment (bartderijk) [inline src/stories/ui-tabs.stories.js:47]: Is the hardcoded 'content' there on purpose?

Comment (dine) [reply src/stories/ui-tabs.stories.js:47]: it's just as an example

Comment (bartderijk) [reply src/stories/ui-tabs.stories.js:47]: Of course, don't mind me, I'll go make some more coffee 😴

**Review by bartderijk — APPROVED** [review]

**Pull request merged**